content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -20,9 +20,9 @@ cluster. This can help to reduce costs and improve performance.
 
 ## Using Service Internal Traffic Policy
 
-Once you have enabled the `ServiceInternalTrafficPolicy`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/),
-you can enable an internal-only traffic policy for a
+Using beta `ServiceInternalTrafficPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is already enabled by default,
+or you can enable an internal-only traffic policy for a
```

Comment:
I don't think this revision is correct. The feature gate is enabled by default.
Only when the feature gate is enabled, you can enable an internal-only traffic policy.
There is no "or" logic here.